client/src/components/FileStatusIcon/FileStatusIcon.scss

```diff
@@ -0,0 +1,35 @@
+.file-status-icon {
+  display: inline-block;
+  margin-left: 6px;
+
+  &:last-of-type {
+    margin-right: 6px;
+  }
```

Comment:
If we did that then we'd need to add more stuff in JS to detect if an FileStatusIcon is that last.  Since we're not using a FileStatusIconList we would end up with duplicated code since this would need to happen in GalleryView.js, TableView.js, Breadcrumb.js, as well as in File/FolderFormFactory.php which doesn't use react.

Much simpler just to keep as is.

Reply:
The key thing I'm asking here is that instead of overriding these values in some places, you remove them from here and instead only introduce them in the places they're needed, whether that's via a modifier class or using the approach you have in other places (wrapping the `.file-status-icon` selector in a class representing the particular area the tweak needs to take place). That means _less_ code, and more importantly less overrides. You can still make use of `:last-of-type`, and you don't need any additional JS logic to make this happen.

Reply:
Have removed margins and added where needed